app/src/components/v-table/table-header/table-header.vue

```diff
@@ -65,9 +67,9 @@ export default defineComponent({
 			type: Object as PropType<Sort>,
 			required: true,
 		},
-		showSelect: {
-			type: Boolean,
-			default: false,
+		selectionMode: {
+			type: String as PropType<LayoutProps['selectionMode']>,
```

Comment:
Maybe we should introduce a separate `SelectionMode` type to separate the table component from layout related types.

This also applies to a few cases below.